tests/ha/check_after_reboot.pm

```diff
@@ -97,6 +97,12 @@ sub run {
     # And check for the state of the whole cluster
     check_cluster_state;
 
+    if (is_sles4sap && check_var('CLUSTER_NAME', 'hana')) {
```

Comment:
I usually did `if (check_var('CLUSTER_NAME', 'hana')) {` without `is_sles4sap` in case of a HANA cluster test on regular SLES. AFAIK it's supported but we don't test it. Do you think that I have to change the current code to restrict it to SLES4SAP?

Reply:
Changing this for consistency. Thanks!